data/tribes/initialization/empire/units.lua

```diff
@@ -1268,20 +1343,34 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_vineyard",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Vineyard
+            lore = pgettext("empire_building",
+               "‘Nunc vino pellite curas!’<br>(Now drive away your cares with wine!)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Vineyard
+            lore_author = pgettext("empire_building", "Imperial drinking song"),
```

Comment:
I think I'll replace this with:
"Vino vendibili hedera non opus est." / "Good wine needs no bush."